project.yml

```diff
@@ -130,4 +138,29 @@ targets:
         script: |
                 # Lava uses the epochMinute as buildNumber
                 /usr/libexec/PlistBuddy -c "Set CFBundleVersion $(($(date +"%s")/60))" ".generated/WaiterRobotLava.plist"
-  
+        
+  WaiterRobotTests:
+    type: bundle.unit-test
+    platform: iOS
+    sources:
+      - WaiterRobotTests
+    dependencies:
+      - target: WaiterRobotLava
+    settings:
+      base:
+        GENERATE_INFOPLIST_FILE: true
+        TEST_TARGET_NAME: WaiterRobotLava
+        TEST_HOST: "$(BUILT_PRODUCTS_DIR)/WaiterRobot Lava.app/$(BUNDLE_EXECUTABLE_FOLDER_PATH)/WaiterRobot Lava"
+
+  WaiterRobotUITests:
+    type: bundle.ui-testing
+    platform: iOS
+    sources:
+      - WaiterRobotUITests
+    dependencies:
+      - target: WaiterRobotLava
+    settings:
+      base:
+        GENERATE_INFOPLIST_FILE: true
+        IPHONEOS_DEPLOYMENT_TARGET: '15.0'
```

Comment:
This is already defined as base (line 26), so it is not needed here again, right?